tests/integration/create_get_delete/integration_test.go

```diff
@@ -12,57 +12,77 @@ import (
 	"time"
 
 	awseks "github.com/aws/aws-sdk-go/service/eks"
+
 	harness "github.com/dlespiau/kube-test-harness"
 	"github.com/dlespiau/kube-test-harness/logger"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gexec"
+
+	"github.com/weaveworks/eksctl/pkg/utils"
+
 	"github.com/weaveworks/eksctl/pkg/testutils/aws"
 	. "github.com/weaveworks/eksctl/pkg/testutils/matchers"
 	"github.com/weaveworks/eksctl/tests/integration"
 	//"k8s.io/client-go/tools/clientcmd"
 )
 
 const (
-	clusterName         = "int-cluster"
-	createTimeoutInMins = 20
-	eksRegion           = "us-west-2"
+	createTimeout = 20
+	region        = "us-west-2"
 )
 
 var (
-	pathToEksCtl      string
-	autoGeneratedConf bool
+	eksctlPath string
+
+	// Flags to help with the development of the integration tests
+	clusterName    string
+	doCreate       bool
+	doDelete       bool
+	kubeconfigPath string
 
-	// Flags to help with developing the integration tests
-	skipCreation    bool
-	skipCleanup     bool
-	pathKubeCtlConf string
+	kubeconfigTemp bool
 )
 
 func TestCreateIntegration(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Integration - Create Suite")
 }
 
+type tInterface interface {
```

Comment:
I like how you did this. I did a similar change but in a fork of github.com/dlespiau/kube-test-harness.

Reply:
Yeah, I have https://github.com/dlespiau/kube-test-harness/pull/22